bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaActionGroupHandler.java

```diff
@@ -15,9 +15,9 @@
 import static org.openhab.binding.somfytahoma.internal.SomfyTahomaBindingConstants.EXECUTE_ACTION;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
 import org.openhab.core.library.types.OnOffType;
-import org.openhab.core.thing.ChannelUID;
-import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.*;
```

Comment:
Please don't use star imports. Update your IDE settings.

Reply:
Fixed

Reply:
It's still there.

Reply:
you are right. now it is fixed